tests/functional/disaster-recovery/metro-dr/test_multiple_apps_failover_and_relocate.py

```diff
@@ -0,0 +1,152 @@
+            if ocs_version <= version.VERSION_4_12:
+                logger.error(
+                    "ODF/ACM version isn't supported for Failover/Relocate operation"
+                )
+                raise NotImplementedError
+
+        acm_obj = AcmAddClusters()
+        primary_instances = []
+        secondary_instances = []
+
+        if workload_type == constants.SUBSCRIPTION:
+            primary_instances, secondary_instances = dr_workloads_on_managed_clusters(
+                num_of_subscription=2, primary_cluster=True, secondary_cluster=True
+            )
+
+        # Set Primary managed cluster
+        set_current_primary_cluster_context(
+            primary_instances[0].workload_namespace, workload_type
+        )
+        self.primary_cluster_name = get_current_primary_cluster_name(
+            namespace=primary_instances[0].workload_namespace,
+            workload_type=workload_type,
+        )
+
+        # Set secondary managed cluster
+        secondary_cluster_name = get_current_secondary_cluster_name(
+            namespace=primary_instances[0].workload_namespace,
+            workload_type=workload_type,
+        )
+
+        # Fence the primary managed cluster
+        enable_fence(drcluster_name=self.primary_cluster_name)
+
+        # Application Failover to Secondary managed cluster
+        config.switch_acm_ctx()
+        for instance in primary_instances:
+            if (
+                config.RUN.get("mdr_failover_via_ui")
+                and workload_type == constants.SUBSCRIPTION
+            ):
+                logger.info(
+                    "Start the process of Failover of subscription based app from ACM UI"
+                )
+                failover_relocate_ui(
+                    acm_obj,
+                    workload_to_move=f"{instance.app_name}-1",
+                    policy_name=instance.dr_policy_name,
+                    failover_or_preferred_cluster=secondary_cluster_name,
+                )
+            else:
+                failover(
+                    failover_cluster=secondary_cluster_name,
+                    namespace=f"{instance.workload_namespace}",
+                    workload_type=workload_type,
+                )
+
+        # Verify application are running in other managedcluster
+        # And not in previous cluster
+        set_current_primary_cluster_context(
+            primary_instances[0].workload_namespace, workload_type
+        )
+        for instance in primary_instances:
+            wait_for_all_resources_creation(
+                instance.workload_pvc_count,
+                instance.workload_pod_count,
+                instance.workload_namespace,
+            )
+
+        # Verify the failover status from UI
+        if config.RUN.get("mdr_failover_via_ui"):
+            config.switch_acm_ctx()
+            verify_failover_relocate_status_ui(acm_obj)
+
+        # TODO Relocate of sub apps
```

Comment:
Track TODO or complete it

Reply:
Pls do for the one added by you

Reply:
https://github.com/red-hat-storage/ocs-ci/issues/10365